### DISCUSSION TITLE
Expandable menu

### DISCUSSION THREAD

**Discussion by savergiggio** [Clooos/Bubble-Card#783] [Feature requests]
Hi, is it possible to create an expandable menu with inside custom cards? thanks in advance

**Comment by DieterKoblenz** [comment]
You can mix Bubble card with  custom:expander-card

https://community.home-assistant.io/t/expander-collapsible-card/507279

**Comment by savergiggio** [comment]
Thank you for your reply, it's what i actually did. But i hope in a built-in version in bubble card 😊

**Reply by lesourcil** [reply]
I support this

**Reply by robblids** [reply]
Do you mind sharing how you styled your card? It looks so big on my dashboard doing like this.

**Comment by stomko11** [comment]
I would love this too. Sometimes I want to "hide" few settings for variety of things. Instead of coming up with new pop-up, it would be nice to just be able to use "expander" card. Tap - rows would show - another tap, rows would hide.

**Comment by stomko11** [comment]
After some time passed, I would still love to see this as new additon to a great pallette of bubble cards. As I stated earlier, I don't necessarily want to have everything shown, and I often utilize popup cards. But things can get pretty messy quickly, not to mention if you nest popup within popup.
I just feel that this could be very useful to many people. I know expander card, I am using it. But having everything under one hood would be perfect.

**Reply by Alefriedo** [reply]
I don't know if it is, what you are looking for, but I just posted my [collapsable sub button menu module](https://github.com/Clooos/Bubble-Card/discussions/2277), which I created because I was looking for something similar.
